backend/ormconfig.json

```diff
@@ -6,7 +6,7 @@
   "password": "postgres",
   "database": "foodies",
   "synchronize": true,
-  "dropSchema": true,
+  "dropSchema": false,
```

Comment:
Decided that its best we don't drop the schema whenever we start the application. If local devs want it to then they can set it back to true!

Reply:
Just curious but what does this mean? 😄